constants/armor.go

```diff
@@ -3,12 +3,18 @@ package constants
 
 // Constants for armored data.
 const (
-	DoChecksum         = true
-	ArmorHeaderEnabled = false // can be enabled for debugging at compile time only
-	ArmorHeaderVersion = "GopenPGP " + Version
-	ArmorHeaderComment = "https://gopenpgp.org"
-	PGPMessageHeader   = "PGP MESSAGE"
-	PGPSignatureHeader = "PGP SIGNATURE"
-	PublicKeyHeader    = "PGP PUBLIC KEY BLOCK"
-	PrivateKeyHeader   = "PGP PRIVATE KEY BLOCK"
+	// ArmorChecksumSetting defines the default behavior for adding an armor checksum
+	// to an armored message.
+	//
+	// If set to true, an armor checksum is added to the message.
+	//
+	// If set to false, no armor checksum is added.
+	ArmorChecksumSetting = true
```

Comment:
OK, I understand a bit better now what the intention is :D How about `ArmorChecksumEnabled`, to match the next one?

Other than that, LGTM :+1: